third_party/terraform/utils/healthcare_utils.go

```diff
@@ -233,3 +233,62 @@ func parseHealthcareDicomStoreId(id string, config *Config) (*healthcareDicomSto
 	}
 	return nil, fmt.Errorf("Invalid DicomStore id format, expecting `{projectId}/{locationId}/{datasetName}/{dicomStoreName}` or `{locationId}/{datasetName}/{dicomStoreName}.`")
 }
+
+type healthcareConsentStoreId struct {
+	DatasetId healthcareDatasetId
+	Name      string
+}
+
+func (s *healthcareConsentStoreId) consentStoreId() string {
+	return fmt.Sprintf("%s/consentStores/%s", s.DatasetId.datasetId(), s.Name)
+}
+
+func (s *healthcareConsentStoreId) terraformId() string {
+	return fmt.Sprintf("%s/%s", s.DatasetId.terraformId(), s.Name)
+}
+
+func parseHealthcareConsentStoreId(id string, config *Config) (*healthcareConsentStoreId, error) {
```

Comment:
Is this significantly different from `parseImportId` with the correct formats? Why do we need this